docs/examples/solar-position/plot_sunpath_diagrams.py

```diff
@@ -103,8 +103,7 @@
 
 tz = 'Asia/Calcutta'
 lat, lon = 28.6, 77.2
-times = pd.date_range('2019-01-01 00:00:00', '2020-01-01', closed='left',
-                      freq='H', tz=tz)
+times = pd.date_range('2019-01-01 00:00:00', '2020-01-01', freq='H', tz=tz)
```

Comment:
Did this break with python 3.8 even though the pandas version didn't change?

Reply:
Prior to this PR, we had ReadTheDocs configured to use python 3.7, but we did not specify a particular pandas version to use for the documentation build.  That means RTD had been using the latest version of pandas that was available for python 3.7 (pandas 1.3.5).  With this PR updating the configuration to use python 3.8 but leaving the pandas version unspecified, ReadTheDocs then had access to a newer version of pandas, and in particular one that required these changes (pandas 2.0.3).  

So updating the python version used in the docs build _did_ change the pandas version that gets used, since we aren't telling RTD to use a particular pandas version.

Reply:
Gotcha.  So the equivalent would now be ```inclusive='left'```?  (That may not matter for the example.)

Reply:
Yes, I think so.  And I agree it does not matter for this example, so I opted to simplify and just remove the parameter rather than update to the new name.